Knowledge-Base/Virtiofs-qs.md

```diff
@@ -0,0 +1,255 @@
+# Virtio-fs Quick start
+
+
+# Overview
+
+Virtiofs is a shared file system that lets virtual machines access a directory tree on the host, it is designed to offer local file system semantics and performance.
+More information on the underlying approach is available at [virtio-fs.gitlab.io](https://virtio-fs.gitlab.io/). Virtiofs for Windows is a user mode file system, implemented using [WinFsp framework](https://github.com/billziss-gh/winfsp). Virtiofs consists of VirtIO-powered driver and user-space service based on WinFsp.
+
+This guide includes setup instructions for QEMU and libvirt, and for Windows guests specifically.
+
+# Status
+
+Virtio-FS is a supported feature. For current known issues, visit the [Virtio-FS GitHub issues page](https://github.com/virtio-win/kvm-guest-drivers-windows/issues?q=is%3Aissue%20state%3Aopen%20label%3Avirtio-fs).
+
+
+# Setup
+
+## Host
+
+This section shows how to setup virtiofs device on the host side. Two options are described: libvirt and QEMU.
+
+### libvirt
+
+Add the following domain XML elements to share the host directory /home/user/viofs with the guest:
+
+```xml
+<domain>
+  ...
+  <memoryBacking>
+    <source type='memfd'/>
+    <access mode='shared'/>
+  </memoryBacking>
+  ...
+  <devices>
+    <filesystem type="mount" accessmode="passthrough">
+      <driver type="virtiofs" queue="1024"/>
+      <source dir="/home/user/viofs"/>
+      <target dir="mount_tag0"/>
+      <address type="pci" domain="0x0000" bus="0x06" slot="0x00" function="0x0"/>
+    </filesystem>
+    <filesystem type="mount" accessmode="passthrough">
+      <driver type="virtiofs" queue="1024"/>
+      <source dir="/home/user/viofs"/>
```

Comment:
I suggest using a different source for a different target to show that the user can mount several FSes.